python/pyproject.toml

```diff
@@ -68,6 +68,9 @@ scikit-learn = [
 fugue = {version = "^0.8.1", optional = true}
 requests = "^2.27"
 
+# NLP related dependencies
+pybloom3-richard = {version = "^0.0.1", optional = true}
```

Comment:
Can we move this fork over and rename before taking this dependency?

Reply:
still need the new package whylogs-bloom published. Blocking issue: https://github.com/whylabs/whylogs-bloom/issues/1